### DISCUSSION TITLE
django work with trino

### DISCUSSION THREAD

**Discussion by TidyData** [trinodb/trino-python-client#372] [Q&A]
Can Django work with Trino?

**Comment by yuokada** [comment]
You mean that Django works with trino as officially supported databases, MySQL, PostgreSQL and etc.
https://docs.djangoproject.com/en/4.2/ref/databases/

Or, are you looking for a similar django extension library for trino like `djongo`?
https://github.com/wsvincent/awesome-django#database-connectors
